authors/apps/articles/models.py

```diff
@@ -22,6 +24,8 @@ class Meta:
     slug = models.SlugField(unique=True, editable=False, max_length=140)
     favorited = models.NullBooleanField(default=False)
     favoritesCount = models.IntegerField(default=0)
+    rating_average = models.DecimalField(
```

Comment:
@micahoriaso what scale of rating are we using?

Reply:
Were rating on a scale of 1-5